limitador/src/storage/distributed/mod.rs

```diff
@@ -241,6 +247,17 @@ impl CrInMemoryStorage {
             });
         }
 
+        // process the re-sync requests...
+        {
+            let limits = limits.clone();
+            tokio::spawn(async move {
+                let limits = limits.clone();
```

Comment:
Confused why we `clone()` the `limits` twice here, am I misreading this?

Reply:
yout right.. not needed.